### PR TITLE
Fix wrong source points for RandomThinPlateSpline

### PR DESCRIPTION
**Pull request by kingsj0405**
### Description

Fix wrong source points of `RandomThinPlateSpline` which leads the augmented image to be distorted too much.

#1186 is a related issue.

### Status
**Ready**

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or new feature that would cause existing functionality to change)
- [ ] New tests added to cover the changes
- [ ] Docstrings/Documentation updated

### PR Implementer
This is a small checklist for the implementation details of this PR.

If there are any questions regarding code style or other conventions check out our
[summary](https://github.com/kornia/kornia/blob/master/CONTRIBUTING.rst).

- [x] Did you discuss the functionality or any breaking changes before ?
- [x] **Pass all tests**: did you test in local ? `make test`
- [ ] Unittests: did you add tests for your new functionality ?
- [x] Documentations: did you build documentation ? `make build-docs`
- [x] Implementation: is your code well commented and follow conventions ? `make lint`
- [x] Docstrings & Typing: has your code documentation and typing ? `make mypy`
- [ ] Update notebooks & documentation if necessary

### KorniaTeam
<details>
  <summary>KorniaTeam workflow</summary>

  - [ ] Assign correct label
  - [ ] Assign PR to a reviewer
  - [ ] Does this PR close an Issue? (add `closes #IssueNumber` at the bottom if
        not already in description)

</details>

### Reviewer
<details>
  <summary>Reviewer workflow</summary>

  - [ ] Do all tests pass? (Unittests, Typing, Linting, Documentation, Environment)
  - [ ] Does the implementation follow `kornia` design conventions?
  - [ ] Is the documentation complete enough ?
  - [ ] Are the tests covering simple and corner cases ?

</details>

### CODE REVIEW

**Comment by edgarriba** [comment]
@kingsj0405 is this producing the expected behavior showed in #1186 ?

**Review by edgarriba — APPROVED** [review]

**Review by dkoguciuk — APPROVED** [review]

**Comment by kingsj0405** [comment]
@edgarriba
Yes this image is output with HEAD of this branch

human_face_test

And this image is output with HEAD of master

human_face_test

```
import random
import torch
import matplotlib as mpl
import matplotlib.pyplot as plt
import cv2
import numpy as np
import face_alignment
from typing import *
from torchvision.transforms import ToTensor
from kornia.geometry.conversions import normalize_pixel_coordinates, denormalize_pixel_coordinates
from kornia.geometry.transform import get_tps_transform, warp_image_tps, warp_points_tps
from kornia.augmentation import RandomThinPlateSpline


def show_tps(img, landmark, img_tps, landmark_tps, show_landmark=True):
    fig, axs = plt.subplots(1, 2, figsize=(16,8))
    axs[0].axis('off')
    axs[1].axis('off')
    axs[0].imshow(img[...,::-1], origin='upper')
    if show_landmark:
        axs[0].scatter(landmark[:, 0], landmark[:, 1], marker='+', color='red', s=40)
    axs[1].imshow(img_tps[...,::-1], origin='upper')
    if show_landmark:
        axs[1].scatter(landmark_tps[:, 0], landmark_tps[:, 1], marker='+', color='red', s=40)
    plt.savefig('human_face_test.png')

# initialize transforms and face_landmark_detector
tran = ToTensor()
fa = face_alignment.FaceAlignment(face_alignment.LandmarksType._2D, device='cuda')
aug = RandomThinPlateSpline(p=1.0)

# read image
human_img = cv2.imread('human_face.jpg')
human_img = tran(human_img)[None,:,:,:]
print(human_img.shape, human_img.max(), human_img.min())
B, C, H, W = human_img.shape

# get landmark
landmark = fa.get_landmarks('human_face.jpg')[-1]

# normalize landmark
landmark = tran(landmark)
print(f'[INFO] landmark: {landmark.shape}, {landmark.max()}, {landmark.min()}')
landmark = normalize_pixel_coordinates(landmark, W, H)
print(f'[INFO] landmark-normalized: {landmark.shape}, {landmark.max()}, {landmark.min()}')

# compute transform
image_tps = aug(human_img.clone().detach())
landmark_tps = landmark
print(f'[INFO] landmark_tps: {landmark_tps.shape}, {landmark_tps.max()}, {landmark_tps.min()}')

# denormalize landmark
landmark = denormalize_pixel_coordinates(landmark, W, H)
print(f'[INFO] landmark-denormalized: {landmark.shape}, {landmark.max()}, {landmark.min()}')
landmark_tps = denormalize_pixel_coordinates(landmark_tps, W, H)
print(f'[INFO] landmark_tps-denormalized: {landmark_tps.shape}, {landmark_tps.max()}, {landmark_tps.min()}')

# Show
show_tps(
    human_img.permute(0,2,3,1)[0].numpy(),
    landmark[0].numpy(),
    image_tps.permute(0,2,3,1)[0].numpy(),
    landmark_tps[0].numpy(),
    False)
```
